spiped/README

```diff
@@ -21,9 +21,12 @@ Options:
 		[ipv6::addr]:port
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
```

Comment:
I agree that the example of address formats should be available in the docs for `spipe`.  I'm not certain why spiped has so many different documentation files:
- `/README`
- `/README.md`
- `/spiped/README`
- `/spiped/spiped.1`
- `/spipe/README`
- `/spipe/spipe.1`

My first instinct is to remove the `/*/README` files since that info seems to be the the man-pages, and to rename the top-level README into something else.  I'll discuss this with @cperciva offline (unless he comments in the next 3 hours).